tests/selection.spec.ts

```diff
@@ -454,3 +455,39 @@ async function clickDivider(page: Page, i = 0) {
   const locator = page.locator('divider-block').nth(i);
   await locator.click();
 }
+
+test('Drag to select tagged text, and copy', async ({ page }) => {
```

Comment:
1. `withCtrlOrMeta` is a better fit here, since `Meta+B` won't trigger copy on Linux CI.
2. We don't use a capital letter at the beginning of a test case name :)
3. Suggestion on line break of a test case:

``` ts
test('drag to select tagged text and copy', async ({ page }) => {
  // put them together for init phrase
  await enterPlaygroundRoom(page);
  await initEmptyParagraphState(page);

  // separate statements by asserts, which generally represents a semantical operation
  await focusRichText(page);
  await page.keyboard.insertText('123456789');
  await assertRichTexts(page, ['123456789']);

  await dragBetweenIndices(page, [0, 1], [0, 3]);
  await withCtrlOrMeta(page, async (page: Page) => {
    await page.press('B');
  });
  await dragBetweenIndices(page, [0, 0], [0, 5]);
  await withCtrlOrMeta(page, async (page: Page) => {
    await page.press('B');
  });
  const textOne = await getSelectedTextByQuill(page);
  expect(textOne).toBe('12345');
});
```

The advice here applies other test cases :)

Reply:
Thx! Learned a lot :)